transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransportImpl.kt

```diff
@@ -1,32 +1,19 @@
 package com.genesys.cloud.messenger.transport.core
 
-import com.genesys.cloud.messenger.transport.network.DEFAULT_PING_INTERVAL_IN_SECONDS
 import com.genesys.cloud.messenger.transport.network.DeploymentConfigUseCase
 import com.genesys.cloud.messenger.transport.network.PlatformSocket
 import com.genesys.cloud.messenger.transport.network.ReconnectionHandlerImpl
 import com.genesys.cloud.messenger.transport.network.WebMessagingApi
 import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
-import com.genesys.cloud.messenger.transport.util.DefaultTokenStore
 import com.genesys.cloud.messenger.transport.util.logs.Log
 import com.genesys.cloud.messenger.transport.util.logs.LogTag
 
-/**
- * Client entry point for the Mobile Messenger Transport SDK.
- */
-object MobileMessenger {
+internal class MessengerTransportImpl(private val configuration: Configuration) {
```

Comment:
wonder if MessengerTransportInternal is better name, I guess `impl` is more suitable for cases when there are some protol that class implements.

Reply:
Will use `...Internal`.

Reply:
Maybe even MessengerClientProvider will be a slightly better name. 
Also was thinking if we should move the fetchDeploymentConfig directly to the `MessengerTransport` layer, instead of nesting it in `MessengerTransportImpl`.